stable/artifactory/README.md

```diff
@@ -1,1160 +1,101 @@
 # JFrog Artifactory Helm Chart
 
-**Heads up: Our Helm Chart docs are moving to our main documentation site. For Artifactory installers, see [Installing Artifactory](https://www.jfrog.com/confluence/display/JFROG/Installing+Artifactory).**
+**Our Helm Chart docs have moved to our main documentation site. For JFrog Artifactory installers, see [Installing Artifactory](https://www.jfrog.com/confluence/display/JFROG/Installing+Artifactory).**
```

Comment:
@aaleksan20 should we directly point to helm documentation in wiki ? - https://www.jfrog.com/confluence/display/JFROG/Installing+Artifactory#InstallingArtifactory-HelmInstallation

Reply:
Yes, good idea